pkgs/applications/gis/qgis/unwrapped.nix

```diff
@@ -19,7 +19,7 @@ let
     urllib3
     pygments
     pyqt5
-    sip
+    sip_4
```

Comment:
What about setting this in `all-packages.nix`?

Reply:
Do you mean doing something like this?
```nix
package = callPackage ../path/to/package { sip = sip_4; };
```
I'm not sure how to do that because `sip` is part of `pythonPackages`

Reply:
Yes, that is the idea.

On the other hand, in pythonPackages the default sip is still the 4.x version. It needs to be investigated!

Well, ignore it for now.

Reply:
> On the other hand, in pythonPackages the default sip is still the 4.x version. It needs to be investigated!

With this PR the default version will be 5.x (and, in the future, it should be updated to 6.x).